src/GitVersion.Core.Tests/VersionCalculation/Strategies/VersionInBranchNameBaseVersionStrategyTests.cs

```diff
@@ -68,8 +66,12 @@ public void ShouldNotTakeVersionFromNameOfNonReleaseBranch(string branchName)
 
     [TestCase("release-2.0.0", "2.0.0")]
     [TestCase("release/3.0.0", "3.0.0")]
-    [TestCase("support/lts-2.0.0", "2.0.0")]
```

Comment:
With the new logic, `support/lts-2.0.0` is not parsed as before?

Reply:
Like you know I have introduced the option `version-in-branch-pattern` and it defaults to the following value:

``` csharp
public const string DefaultVersionInBranchPattern = @"(?<version>[vV]?\d+(\.\d+)?(\.\d+)?).*";
```

That means the behavior is stricter then before right and allows only the name after the version number. If the user don't want this he or she can change it in the configuration very easily. But we want to not allow `release/downgrade-some-lib-to-3.2.1`. What is you suggestion?

Reply:
Ok, I agree with the stricter behaviour. We just need to document it, like you have.